db/query.go

```diff
@@ -76,7 +76,8 @@ var QueryChannels = SGQuery{
 			"FROM `%s` "+
 			"USE INDEX ($idx) "+
 			"UNNEST OBJECT_PAIRS($sync.channels) AS op "+
-			"WHERE [op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),IFMISSING(op.val.del,null)]  BETWEEN  [$channelName, $startSeq] AND [$channelName, $endSeq] "+
+			"WHERE $activeOnlyFilter [op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),"+
+			"IFMISSING(op.val.del,null)]  BETWEEN  [$channelName, $startSeq] AND [$channelName, $endSeq] "+
```

Comment:
Can we append $$activeOnlyFilter after this line, instead of prepending immediately after WHERE?  The 'WHERE $activeOnlyFilter [op.name' is confusing at first glance.

Reply:
Appended $$activeOnlyFilter at the end of BETWEEN, instead of prepending immediately after WHERE